### DISCUSSION TITLE
PriceFeedVersionIsAccurate Test is failing!!

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2642] [Q&A] [answered]
![Screenshot (55)](https://github.com/user-attachments/assets/3e73d219-fe33-43c5-8235-d29cbef50b69)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, The test is failing because the `priceFeed` address you are using is not on Anvil. You are supposed to deploy a new priceFeed contract in your deployment process when running on Anvil. If you run the process on `sepolia` it will pass.

**Reply by prakharmishra3956** [reply]
It's still failing 
![Screenshot (56)](https://github.com/user-attachments/assets/7a72a62e-14d2-4e4a-bf4c-a2f99e81ff41)

**Reply by EngrPips** [reply]
Please show the `testPriceFeedVersionIsAccurate` test.

**Reply by prakharmishra3956** [reply]
![Screenshot (58)](https://github.com/user-attachments/assets/62937932-f318-4b80-b314-c2c859ab1019)

**Reply by EngrPips** [reply]
Put `console.log(fundMe)` before your `assertEq(version, 4)` and re-run the test let's see what will be printed to the console.

**Reply by prakharmishra3956** [reply]
![Screenshot (59)](https://github.com/user-attachments/assets/d215ea44-6a76-4ed5-b6d4-f09b4c7f867e)
It's showing error.

**Reply by EngrPips** [reply]
Please import `console` from `forge-std` using the below import code 

```js
import {console} from "forge-std/console.sol";
```
then retry.

**Reply by prakharmishra3956** [reply]
![Screenshot (60)](https://github.com/user-attachments/assets/03b9616b-9316-450b-a9bf-d6f6a2479d07)
Still Same.

**Reply by EngrPips** [reply]
You don't always save your files. Why is that?

**Reply by EngrPips** [reply, marked as answer]
After any changes you make to your file, please make it a habit to always press `cltr + s` before compiling or running any test and use `console.log(address(fundMe))` instead of `console.log(fundMe)`

**Reply by prakharmishra3956** [reply]
thanks, It's done and I will remember saving the files.

**Reply by EngrPips** [reply]
Good
